templates/insurance_companies/general_edit.php

```diff
@@ -24,70 +24,70 @@ function jsWaitForDelay(delay) {
 <!-- it is important that the hidden form_id field be listed first, when it is called it populates any old information attached with the id, this allows for partial edits
         if it were called last, the settings from the form would be overwritten with the old information-->
 <input type="hidden" name="form_id" value="<?php echo $this->insurancecompany->id;?>" />
-<table style="font-size:9pt;" width="500px" CELLSPACING="0" CELLPADDING="3">
+<table class="table table-hover">
 <tr>
-    <td width="220px" VALIGN="MIDDLE" ><?php echo xl("Name");?> </td>
-    <td VALIGN="MIDDLE" >
-        <input type="text" size="40" name="name" value="<?php echo $this->insurancecompany->get_name();?>" onKeyDown="PreventIt(event)" /> 
-        Required
+    <td><?php echo xl("Name");?> </td>
+    <td>
+        <input type="text" class="form-control" size="40" name="name" value="<?php echo $this->insurancecompany->get_name();?>" onKeyDown="PreventIt(event)" /> 
+        *Required
     </td>
 </tr>
 <tr>
-    <td VALIGN="MIDDLE" ><?php echo xl("Attn");?> </td>
-    <td VALIGN="MIDDLE" >
-        <input type="text" size="40" name="attn" value="<?php echo $this->insurancecompany->get_attn();?>" onKeyDown="PreventIt(event)" />
+    <td><?php echo xl("Attn");?> </td>
+    <td>
+        <input type="text" class="form-control" size="40" name="attn" value="<?php echo $this->insurancecompany->get_attn();?>" onKeyDown="PreventIt(event)" />
     </td>
 </tr>
 
 <tr>
-    <td VALIGN="MIDDLE" ><?php echo xl("Address");?> </td>
-    <td VALIGN="MIDDLE" >
-        <input type="text" size="40" name="address_line1" value="<?php $this->insurancecompany->address->line1;?>" 
+    <td><?php echo xl("Address(line1)");?> </td>
```

Comment:
the translation engine will have trouble with the punctuation in side of the statement Maybe something like 
xlt("Address". ( . "line1". ) .") Not real sure about the syntax.